addons/website/tests/test_unsplash_beacon.py

```diff
@@ -16,8 +16,29 @@ def test_01_beacon(self):
             <t t-set="pageName" t-value="'homepage'"/>
             <div id="wrap" class="oe_structure oe_empty">
                 <img src="/unsplash/pYyOZ8q7AII/306/fairy.jpg"/>
+                <script>
+                    Object.defineProperty(window, "$", {
+                        get() {
+                            return this._patched$;
+                        },
+                        set(value) {
+                            delete this.$;
+                            this._patched$ = value;
+                            // Patch RPC call.
+                            const oldGet = value.get.bind(this);
+                            value.get = (url, data, success, dataType) => {
+                                if (url === "https://views.unsplash.com/v") {
+                                    const imageEl = document.querySelector(`img[src^="/unsplash/${data.photo_id}/"]`);
+                                    imageEl.dataset.beacon = "sent";
+                                    return;
+                                }
+                                return oldGet(url, data, success, dataType);
+                            };
+                        },
+                    });
+                </script>
```

Comment:
It's weird to do it like that IMO.. I will see if there is not another way to test the unsplash beacon...

I checked a little bit the issue I cannot make the test fail on Odoo 14.0... I checked the runbot and it looks like it is only failing from Odoo 16.X. Can you check and explain why ?